src/StatusQ/Controls/StatusTagSelector.qml

```diff
@@ -0,0 +1,109 @@
+import QtQuick 2.14
+import QtQuick.Layouts 1.12
+import QtQuick.Controls 2.14
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+
+//TODO check colors, sizes, margins, general clean up
```

Comment:
So.. is this component a work-in-progress?

Reply:
more or less. I need to double check colors, fonts, etc but I believe most of functionality is already there

Reply:
Gotcha!